ajax/libs/jquery.mb.bgndgallery/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jquery.mb.bgndgallery",
+  "filename": "jquery.mb.bgndGallery.min.js",
+  "version": "1.9.5",
+  "description": "This plugin let you create a flexible photo gallery that you can use also as background of your site.",
+  "keywords": [
+    "jquery-plugin",
+    "image",
+    "gallery",
+    "background",
+    "CSS3",
+    "effects",
+    "HTML5"
+  ],
+  "author": "Pupunzi <matteo@open-lab.com> (Matteo Bicocchi)",
+  "license": "MIT",
+  "homepage": "https://pupunzi.open-lab.com/mb-jquery-components/jquery-mb-bgndgallery/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pupunzi/jquery.mb.bgndGallery.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/pupunzi/jquery.mb.bgndGallery.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
Sorry for the late reply, too many notifications by my side 😓 ... It's better to exclude html here.